src/pages/Trades/TokensTrades.tsx

```diff
@@ -53,6 +54,18 @@ export const TokensTradesPage: FC<TokensTradesPage> = ({ currentTab }) => {
     });
   }, [selectedAccount?.address, currentTab, sortString, pageSize]);
 
+  const debouncedFetchTrades = useCallback(() => {
+    return debounce(function (...args: string[]) {
+      fetch({
```

Comment:
1. copy-paste (same code in onSearch)
2. searchValue not being set here (see line 46)

Consider using onSearch function for "onSearchStringChange" and debounce it instead.

Reply:
fixed